src/LogentriesCore.NetCore/LogentriesCore.NetCore.csproj

```diff
@@ -0,0 +1,65 @@
+    <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
+    <PackageReleaseNotes></PackageReleaseNotes>
+    <Copyright>Copyright 2011</Copyright>
+    <PackageTags>Logentries;logging</PackageTags>
+    <PackageProjectUrl>https://github.com/logentries/le_dotnet</PackageProjectUrl>
+
+    <GenerateAssemblyTitleAttribute>false</GenerateAssemblyTitleAttribute>
+    <GenerateAssemblyDescriptionAttribute>false</GenerateAssemblyDescriptionAttribute>
+    <GenerateAssemblyConfigurationAttribute>false</GenerateAssemblyConfigurationAttribute>
+    <GenerateAssemblyCompanyAttribute>false</GenerateAssemblyCompanyAttribute>
+    <GenerateAssemblyCopyrightAttribute>false</GenerateAssemblyCopyrightAttribute>
+    <GenerateAssemblyFileVersionAttribute>false</GenerateAssemblyFileVersionAttribute>
+    <GenerateAssemblyProductAttribute>false</GenerateAssemblyProductAttribute>
+    <GenerateAssemblyVersionAttribute>false</GenerateAssemblyVersionAttribute>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="..\LogentriesCore\AsyncLogger.cs" Link="AsyncLogger.cs" />
+    <Compile Include="..\LogentriesCore\LeClient.cs" Link="LeClient.cs" />
+    <Compile Include="..\LogentriesCore\Properties\AssemblyInfo.cs" Link="AssemblyInfo.cs" />
+    <Compile Include="..\LogentriesCore\SettingsLookup.cs" Link="SettingsLookup.cs" />
+    <Compile Include="..\LogentriesCore\SettingsLookupFactory.cs" Link="SettingsLookupFactory.cs" />
+  </ItemGroup>
+
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net35' ">
+    <Reference Include="System" />
+    <Reference Include="System.Configuration" />
+    <PackageReference Include="TaskParallelLibrary" Version="1.0.2856.0" />
+  </ItemGroup>
+
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net40' ">
+    <Reference Include="System" />
+    <Reference Include="System.Configuration" />
+    <PackageReference Include="Microsoft.WindowsAzure.ConfigurationManager" Version="3.1.0" />
+  </ItemGroup>
+
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard1.3' ">
+    <PackageReference Include="Microsoft.Extensions.Configuration" Version="1.0.2" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.EnvironmentVariables" Version="1.0.2" />
+    <PackageReference Include="System.Diagnostics.TraceSource" Version="4.0.0" />
+    <PackageReference Include="System.Net.Security" Version="4.0.1" />
+    <PackageReference Include="System.Threading" Version="4.0.11" />
+    <PackageReference Include="System.Threading.Thread" Version="4.0.0" />
+  </ItemGroup>
+
+  <PropertyGroup Condition=" '$(TargetFramework)' == 'netstandard1.3' ">
+    <DefineConstants>$(DefineConstants);NETSTANDARD1_3</DefineConstants>
+  </PropertyGroup>
+
+  <PropertyGroup Condition=" '$(TargetFramework)' == 'net35' ">
+    <DefineConstants>$(DefineConstants);NET3_5</DefineConstants>
+  </PropertyGroup>
```

Comment:
These seem redundant; `Microsoft.NET.SDK` already defines these constants: dotnet/sdk#309.

Note per SDK defined constant:
'net35' => NET35  (instead of NET3_5)